### PR TITLE
Alarm and Actions Tests

### PR DESCRIPTION
**Pull request by molly-moen**
# Description and Motivation 
Add tests to check if alarms are generated, and if alarm and rule scenarios work with and without actions configured in a rule. These changes depend on actions updates, which are currently in a feature branch

# Change type 
- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: Telemetry/AlarmsTest.cs
Comment (zhang-hua) [inline Telemetry/AlarmsTest.cs]: nit: should we not using hardcode device ids here? Maybe we can improve it later.

**Review by zhang-hua — APPROVED** [review]

**Comment by molly-moen** [comment]
holding this PR until actions code is merged to master

**Pull request merged**
